docs/input-manifest/providers/aws.md

```diff
@@ -138,17 +138,17 @@ spec:
         serverType: t3.medium
         # AMI ID of the image.
         # Make sure to update it according to the region. 
-        image: ami-0965bd5ba4d59211c
+        image: ami-0618becf24e8d2b5f
         storageDiskSize: 50
 
       - name: compute-2-aws
         providerSpec:
           # Name of the provider instance.
           name: aws-1
           # Region of the nodepool.
-          region: eu-central-3
+          region: eu-central-3 # this is just an example, it does not really exists!
```

Comment:
```suggestion
          region: eu-central-2
```

Reply:
Could we just change this to a region that exists, then? What do you think @Despire ?

Reply:
Yes, this was probably a typo in the docs

Reply:
I'll commit this suggestion and resolve conversation then